Samples/FilterOnObservableCondition/MainView.js

```diff
@@ -35,11 +35,11 @@ var items = Observable(
 	new Color('#FFFF00', 'Other 5', COLOR_TYPES.OTHER)
 );
 
-var filteredItems = currentCategory.map(function(category){
```

Comment:
See for yourself ;)

https://github.com/fusetools/fuselibs/blob/db8c54a4f663670812a02e2f9a90d9a3344a1a65/Source/Fuse.Reactive/FuseJS/Observable.js#L754